coral/src/domain/topic/topic-types.ts

```diff
@@ -19,6 +20,11 @@ type NoContent = {
   status: boolean;
 };
 
+type TopicType = "Consumer" | "Producer";
```

Comment:
I see that might have been where the interrogation about that kind of type came from haha! It odes look a little weird like this. Would it be good replaced by something like this when the backend changes are merged?

```ts
type TopicType = KlawApiRequestQueryParameters<"getTopics">["topicType"]
```

Reply:
I will update it later, yes! At the moment, it's just a `string` so I wanted it to be more specific.